netgames/entropy/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ set(CPPS EntropyBase.cpp EntropyPackets.cpp EntropyRoom.cpp)
 set(NETGAME_MODULE entropy)
 
 add_library(${NETGAME_MODULE} MODULE ${CPPS} ${HEADERS})
+set_target_properties(${NETGAME_MODULE} PROPERTIES CXX_VISIBILITY_PRESET "hidden")
```

Comment:
I'm thinking about writing a custom CMake command for netgames that are all built the same way, maybe for a future contribution